server/runtime/storage/influxdb/index.js

```diff
@@ -112,7 +112,7 @@ function Influx(_settings, _log) {
             } else {
                 const point = new Point(tag.id)
                     .tag('id', tag.id)
-                    .tag('name', tag.name)
+                    .tag('name', tag.tagref.name)
```

Comment:
Hi thanks for the PR, I'm not sure if all devices set the tagref, what do you think about my suggestion
`.tag('name', tag.name || tag.tagref.name)`

Reply:
perfect